bridges/otelzap/core_test.go

```diff
@@ -174,3 +180,80 @@ func TestConvertLevel(t *testing.T) {
 		}
 	}
 }
+
+// Benchmark with multiple Fields.
+func BenchmarkMultipleFields(b *testing.B) {
```

Comment:
This is mainly about testing the `Write` method

```suggestion
func BenchmarkCoreWrite(b *testing.B) {
```